membership-attribute-service/app/models/Attributes.scala

```diff
@@ -60,8 +61,9 @@ case class Attributes(
   lazy val isPaidTier = isSupporterTier || isPartnerTier || isPatronTier || isStaffTier
   lazy val isAdFree = AdFree.exists(identity)
   lazy val isContributor = RecurringContributionPaymentPlan.isDefined
+  lazy val digitalSubscriberHasActivePlan = DigitalSubscriptionExpiryDate.exists(_.isAfter(now))
 
-  lazy val contentAccess = ContentAccess(member = isPaidTier || isFriendTier, paidMember = isPaidTier, recurringContributor = isContributor) // we want to include staff!
+  lazy val contentAccess = ContentAccess(member = isPaidTier || isFriendTier, paidMember = isPaidTier, recurringContributor = isContributor, digitalPack = digitalSubscriberHasActivePlan || isStaffTier)
```

Comment:
Why does isStaffTier = true imply digitalPack should be true?

Reply:
I think it'd be nice if all staff got access to the Daily Edition, it saves us generating urgent override tokens. Also we can get more feedback to make the product even better! I'll get this signed off before the new app gets published on the Apple App Store.

Reply:
it'd be sensible if staff get everything that doesn't cost us to fulfil, but I don't know the policy on that.